src/main/scala/org/camunda/feel/impl/builtin/StringBuiltinFunctions.scala

```diff
@@ -172,6 +174,23 @@ object StringBuiltinFunctions {
     }
   )
 
+  private def extractFunction = builtinFunction(
+    params = List("input", "pattern"),
+    invoke = {
+      case List(ValString(input), ValString(pattern)) => {
+        val p = Pattern.compile(pattern)
```

Comment:
Please have a look at the latest changes on master. For the other functions, I wrapped the creation of the pattern into a `Try` block to avoid exceptions in case of invalid patterns.